front_end_pocket_poll/lib/groups_widgets/groups_home.dart

```diff
@@ -571,20 +571,45 @@ class _GroupsHomeState extends State<GroupsHome>
             child: Icon(Icons.add),
             key: Key("groups_home:new_group_button"),
             onPressed: () {
-              // Navigate to second route when tapped.
-              Navigator.push(
-                context,
-                MaterialPageRoute(builder: (context) => GroupCreate()),
-              ).then((val) {
-                refreshList();
-              });
+              if (Globals.user.ownedGroupsCount < Globals.maxOwnedGroups) {
+                Navigator.push(
+                  context,
+                  MaterialPageRoute(builder: (context) => GroupCreate()),
+                ).then((val) {
+                  refreshList();
+                });
+              } else {
+                // we aren't allowing users to go to this page if they have no categories, so show a popup telling them this
+                print(Globals.user.ownedGroupsCount);
```

Comment:
Remove print

Reply:
removed and updated the incorrect comment above this.